presto-docs/src/main/sphinx/functions/ip.rst

```diff
@@ -49,3 +49,13 @@ IP Functions
         SELECT is_subnet_of(IPPREFIX '192.168.3.131/26', IPPREFIX '192.168.3.144/30'); -- true
         SELECT is_subnet_of(IPPREFIX '64:ff9b::17/64', IPPREFIX '64:ffff::17/64'); -- false
         SELECT is_subnet_of(IPPREFIX '192.168.3.131/26', IPPREFIX '192.168.3.131/26'); -- true
+
+.. function:: ip_prefix_agg(array(ip_prefix)) -> array(ip_prefix)
+
+    Returns the minimal CIDR representation of the input ``IPPREFIX`` array.
+    All ``IPPREFIX``s in the input array must have the same IP version (i.e., only IPv4 or only IPv6). ::
```

Comment:
```suggestion
    Every ``IPPREFIX`` in the input array must be the same IP version (that is, only IPv4 or only IPv6). ::
```
* Adding an `s` after the ``IPPREFIX`` here causes the following error in a local doc build: 

  `/Users/steveburnett/Documents/GitHub/presto/presto-docs/src/main/sphinx/functions/ip.rst:55: WARNING: Inline literal start-string without end-string.` 
  Suggest rephrase to avoid the warning. 

* Do not use Latin abbreviations. See the [GitLab documentation style guide recommended word list entry for i.e.](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#ie).